app/views/news/show.html.haml

```diff
@@ -0,0 +1,11 @@
+.container
+  %h1= @news.title
+  %hr
+  %span= t('.posted_on')
```

Comment:
why do you use span here?

Reply:
I want that date of news creation must be in one line, like this : Posted at April 21, 2018 11:02
not like: 
Posted at
 April 21, 2018 11:02

Reply:
ok